RFCs/FS-1032-fsharp-in-dotnet-sdk.md

```diff
@@ -133,6 +133,133 @@ Relevant comment from @dsyme
 > 
 > I was so excited by this result I told my wife.  That’s a sure indicator that we have to make sure we keep that value. 
 
+### DotNet SDK F# Templates 
+
+https://github.com/fsharp/fslang-design/blob/master/RFCs/FS-1032-fsharp-in-dotnet-sdk.md
+The basic template for a console app looks like:
+````
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Program.fs" />
+  </ItemGroup>
+</Project>
+````
```

Comment:
Super - please include that information then, thanks.  It's important to call that out :)